Sources/Sonar/APIs/AppleRadarRouter.swift

```diff
@@ -59,7 +60,7 @@ enum AppleRadarRouter {
                     "configurationSplitValue": radar.configuration,
                     "workAroundText": "",
                     "descriptionText": radar.body,
-                    "problemAreaTypeCode": radar.area.map { String($0.appleIdentifier) } ?? "",
+                    "problemAreaTypeCode": (radar.area.map { String($0.appleIdentifier) } ?? "") as Any,
```

Comment:
Yep, this was introduced by my sizes ternary below. Here's a pretty small repro case:

```swift
struct Area {
    let appleIdentifier: Int
}

struct Radar {
    let area: Area?

    init() {
        self.area = nil
    }
}

let radar = Radar()
let sizes = [""]
let JSON: [String: Any] = [
    "problemAreaTypeCode": radar.area.map { String($0.appleIdentifier) } ?? "",
    "hiddenFileSizeNew": [].isEmpty ? "" : sizes,
]
```